src/pages/en/guides/integrations-guide/vercel.md

```diff
@@ -32,7 +32,7 @@ First, install the `@astrojs/vercel` package using your package manager. If you'
 npm install @astrojs/vercel
 ```
 
-Then, install this adapter in your `astro.config.*` file using the `adapter` property (note the import from `@astrojs/vercel/serverless` - see [targets](https://github.com/withastro/astro/tree/main/packages/integrations/vercel/#targets)).
+Then, install this adapter in your `astro.config.*` file using the `deploy` property (note the import from `@astrojs/vercel/serverless` - see [targets](https://github.com/withastro/astro/tree/main/packages/integrations/vercel/#targets)).
```

Comment:
I think this may have snuck through erroneously upstream, so putting this PR on hold until we have confirmation.